changelog.d/4384.feature

```diff
@@ -0,0 +1,2 @@
+MXRoomSummaryStore: Implementation with Core Data.
+MXRoomListDataManager: Implementation with Core Data.
```

Comment:
Will Towncrier read this second line? (I'm interested to find out 🧐)

From the [readme](https://github.com/matrix-org/matrix-ios-sdk/blob/develop/CONTRIBUTING.md#changelog)
> The file must only contain one line.

Reply:
+1. We need one line per new feature